client/blocks/plan-storage/test/bar.jsx

```diff
@@ -0,0 +1,121 @@
+/** @format */
+
+jest.mock( 'lib/abtest', () => ( {
+	abtest: () => '',
+} ) );
+
+const translate = x => x;
+jest.mock( 'i18n-calypso', () => ( {
+	localize: Comp => props => (
+		<Comp
+			{ ...props }
+			translate={ function( x ) {
+				return x;
+			} }
+		/>
+	),
+	numberFormat: x => x,
+} ) );
+
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
```

Comment:
I'm really happy to see more tests being added, thanks for that!

Are you familiar with the [testing docs](https://github.com/Automattic/wp-calypso/blob/master/docs/testing/testing-overview.md#faq) for the repo? There's been some work to migrate existing tests to [Jest `expect`](https://facebook.github.io/jest/docs/en/expect.html), it would be great if new tests used it.

Reply:
Good call, I missed that. I'll use Jest expect then :)